src/containers/StructurePage/resourceComponents/AllResourcesGroup.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { Plus } from '@ndla/icons/action';
+import BEMHelper from 'react-bem-helper';
+import AddTopicResourceButton from './AddTopicResourceButton';
+import Accordion from '../../../components/Accordion';
+import ResourceItems from './ResourceItems';
+import AddResourceModal from './AddResourceModal';
+import { RESOURCE_TYPE_LEARNING_PATH } from '../../../constants';
+
+import { Filter } from '../../../interfaces';
+
+export const classes = new BEMHelper({
+  name: 'topic-resource',
+  prefix: 'c-',
+});
+
+interface Props {
+  topicResources: {
+    resources: [];
+    contentType: string;
+  }[];
+  resourceTypes: {
+    id: string;
+    name: string;
+    subtypes: [];
+    disabled: boolean;
+  }[];
+  params: {
+    topic: string;
+    subtopics: string;
+  };
+  refreshResources: () => void;
+  availableFilters: Filter;
+  activeFilter: string;
+  locale: string;
+  currentTopic: {
+    filters: Filter[];
+  };
+  currentSubject: {
+    id: string;
+    name: string;
+  };
+  structure: {}[];
+}
+
+const ResourceGroup = ({
+  resourceTypes,
+  topicResources,
+  t,
+  params,
+  refreshResources,
+  availableFilters,
+  activeFilter,
+  locale,
+  currentTopic,
+  currentSubject,
+  structure,
+}: Props & tType) => {
+  const [displayResource, setDisplayResource] = useState<boolean>(false);
+  const [showAddModal, setShowAddModal] = useState<boolean>(false);
+
+  const toggleDisplayResource = () => {
+    setDisplayResource(prev => !prev);
+  };
+
+  const toggleAddModal = () => {
+    setShowAddModal(prev => !prev);
+  };
+
+  const topicId = params.subtopics?.split('/')?.pop() || params.topic;
```

Comment:
Bartek lagde en helper for å hente ut denne iden som du kan bruke.

Reply:
Sure, denne fila er kopiert fra ResourceGroup forresten som du sikkert skjønner

Reply:
tenker du på getCurrentTopic?

Reply:
nei getIdFromUrn i taxonomyHelpers.

Reply:
den ser annerledes ut enn det som gjøres her

Reply:
Ja, eg blingsa.